components/eam/bld/namelist_files/namelist_definition.xml

```diff
@@ -4941,6 +4941,12 @@ Seasalt aerosol emission tuning factor
 Default: set by build-namelist.
 </entry>
 
+<entry id="dms_emis_scale" type="real" category="cam_chem"
+       group="chem_inparm" valid_values="" >
+DMS emission tuning factor
+Default: set by build-namelist.
```

Comment:
Thanks for working on this PR, @mahf708. All look good. Below more of an FYI.

Need to also call add_default($nl, 'dms_emis_scale') in eam/bld/build-namelist in order to pick up the value from namelist_defaults_eam.xml. Alternatively, we can keep as is for now, but change "set by build-namelist" to "1.0 as hardwired default (without seeing a value in atm_in).  With that, alternative value can still be specified in user_nl_eam.

If a scaling factor is recommended for the eventual v3, we will need that add_default in build-namelist.